torch/csrc/jit/ir.h

```diff
@@ -139,6 +142,24 @@ struct Scope {
     }
     return current;
   }
+  bool hasConstant(std::string name) {
+    return constants_.find(name) != constants_.end();
+  }
+  int getConstant(std::string name) {
+    return constants_[name];
+  }
+  void addConstant(std::string name, int value) {
+    constants_[name] = value;
+  }
+  bool hasNameAlias(std::string alias) {
+    return input_aliases_.find(alias) != input_aliases_.end();
+  }
+  std::string getNameAlias(std::string alias) {
```

Comment:
You're taking the `std::string` by value and making a copy each time. I think you want `const std::string&`!